packages/manager/cypress/e2e/core/databases/create-database.spec.ts

```diff
@@ -118,8 +194,24 @@ describe('create a database cluster, mocked data', () => {
           `/databases/${databaseMock.engine}/${databaseMock.id}`
         );
 
-        cy.findByText(databaseMock.label).should('be.visible');
-        cy.findByText(databaseRegionLabel).should('be.visible');
+        // Validate Cluster Configuration on Summary page
+        cy.wrap([
+          'Status',
+          'Plan',
+          'Nodes',
+          'CPUs',
+          'Engine',
+          'Region',
+          'RAM',
+          'Total Disk Size',
+          databaseMock.label,
+          databaseRegionLabel,
+          `${configuration.engine} v${configuration.version}`,
+          nodes,
+          `${databaseMock.total_disk_size_gb} GB`,
+        ]).each((text: string) => {
+          cy.findByText(text).should('be.visible');
+        });
```

Comment:
```suggestion
        [
          'Status',
          'Plan',
          'Nodes',
          'CPUs',
          'Engine',
          'Region',
          'RAM',
          'Total Disk Size',
          databaseMock.label,
          databaseRegionLabel,
          `${configuration.engine} v${configuration.version}`,
          nodes,
          `${databaseMock.total_disk_size_gb} GB`,
        ].forEach((text: string) => {
          cy.findByText(text).should('be.visible');
        });
```
Just a small nit: there's no need to use `cy.wrap` here, we can just loop through the array directly.